device/dhcp/common/dhcpv6options.yaml

```diff
@@ -0,0 +1,387 @@
+        The encapsulated vendor-specific options field is encoded as a sequence of code/length/value fields of 
+        identical format to the DHCP options field. The option codes are defined by the vendor identified in the 
+        enterprise-number field and are not managed by IANA.
+      type: object
+      properties:
+        code:
+          description: >-
+            The code for the encapsulated option.
+          type: integer
+          format: uint32
+          x-field-uid: 1  
+        data:
+          description: >-
+            The data for the encapsulated option.
+          type: string
+          x-field-uid: 2
+
+    Dhcpv6Options.Fqdn:
+      description: >-
+        DHCPv6 server needs to know the FQDN of the client for the addresses for the client's IA_NA bindings in order to
+        update the IPv6-address-to-FQDN mapping. This option allows the client to convey its FQDN to the server. The Client 
+        FQDN option also contains Flags that DHCPv6 clients and servers use to negotiate who does which updates.
+      type: object
+      properties:
+        flag_s:
+          description: >-
+            The "S" bit indicates whether the server should or should not perform the AAAA RR (FQDN-to-address) DNS updates. 
+            A client sets the bit to 0 to indicate that the server should not perform the updates and 1 to indicate that the 
+            server should perform the updates. The state of the bit in the reply from the server indicates the action to be 
+            taken by the server. If it is 1, the server has taken responsibility for AAAA RR updates for the FQDN.
+          type: boolean
+          default: true
+          x-field-uid: 1
+        flag_o:
+          description: >-
+            The "O" bit indicates whether the server has overridden the client's preference for the "S" bit. A client must set this 
+            bit to 0. A server must set this bit to 1 if the "S" bit in its reply to the client does not match the "S" bit received 
+            from the client.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        flag_n:
+          description: >-
+            The "N" bit indicates whether the server should not perform any DNS updates. A client sets this bit to 0 to request that 
+            the server should perform updates (the PTR RR and possibly the AAAA RR based on the "S" bit) or to 1 to request that the 
+            server should not perform any DNS updates. A server sets the "N" bit to indicate whether the server shall (0) or shall not (1) 
+            perform DNS updates. If the "N" bit is 1, the "S" bit MUST be 0.
+          type: boolean
+          default: false
+          x-field-uid: 3
+        domain_name:
+          description: >-
+            The Domain Name part of the option carries all or part of the FQDN of a DHCPv6 client. A client MAY also leave the 
+            Domain Name field empty if it desires the server to provide a name.
```

Comment:
Maybe example format fo FQDN and url below might be helpful for user

Reply:
added